### DISCUSSION TITLE
What important data stored in /atlantis-data directory

### DISCUSSION THREAD

**Discussion by sumodirjo** [runatlantis/helm-charts#432] [General]
Hi,
I have atlantis deployment in GKE using helm-chart version 4.19. The current PV mounted to /atlantis-data is using pd-standard disk. I plan to change it to pd-ssd (premium-rwo storageclass).

I checked the directory contain terraform plugin, repository contain tf code and atlantis.db.

Reading around my understanding is that atlantis.db contain terraform plan. I want to ask, are there any important data in atlantis.db file? is it ok if I recreate the disk and not restore atlantis.db file?

Thank you.

**Comment by GMartinez-Sisti** [comment]
Hi @sumodirjo,

The Atlantis database is used to store the existing state of the terraform plans and Pull Request/Merge Request locks. If you lose it, in the worst case scenario you will need to plan everything again and the locks will be gone.

The Atlantis data directory stores all the downloaded repository files from the planned branch and the downloaded providers that were required for the terraform run.

Depending on the size of your org losing these might not be a big deal. We had a breaking change that required changing the Persistent Volumes, take a look and see if it can get you some ideas for a migration without loosing the data -> https://github.com/runatlantis/helm-charts/blob/main/charts/atlantis/README.md#from-4-to-5. Otherwise you just start from scratch with the new PV.

Hope it helps.
